plugins/nerdicons.lua

```diff
@@ -121,7 +121,7 @@ local known_filenames_icons = {
   [".gitignore"]      = { "#cc3e55", "dev-git"              },
   [".gitmodules"]     = { "#cc3e56", "dev-git"              },
   ["PKGBUILD"]        = { "#6d8ccc", "md-package"           },
-  ["license"]         = { "#d0bf41", "seti-license"         },
+  ["license"]         = { "#d0bf41", "seti-license"         }, ["copying"] = { "#d0bf41", "seti-license" },

```

Comment:
```suggestion
  ["license"]         = { "#d0bf41", "seti-license"         },
  ["copying"]         = { "#d0bf41", "seti-license"         },
```

Reply:
Are you sure this is the way you want to do it? Why are the two plugins formatted differently?

Reply:
Someone decided to format the `nerdicons` plugin, so here we are.

We might format the `nonicons` one in the future, but no one did it yet.

Reply:
Okay. I also bumped the version number.